packages/pancake-uikit/src/widgets/Menu/statusConsts.ts

```diff
@@ -0,0 +1,3 @@
+export const LIVE = { text: "LIVE", color: "failure" };
```

Comment:
this could go in the config file, with an object `Status`

Reply:
Added that in here, see if you're happy with the implementation? https://github.com/pancakeswap/pancake-toolkit/pull/29/files#diff-93fe9389e18f0ecc161575820214af1f993d1c81688a0f2a24d6c9bca4fb9628R1-R17

Reply:
Tweaked this and exported them. https://github.com/pancakeswap/pancake-toolkit/pull/29/files#diff-93fe9389e18f0ecc161575820214af1f993d1c81688a0f2a24d6c9bca4fb9628R3-R16